terraform/modules/self-service/ecs.tf

```diff
@@ -18,7 +18,7 @@ locals  {
     sentry_dsn                         = "arn:aws:ssm:eu-west-2:${data.aws_caller_identity.account.account_id}:parameter/${var.deployment}/${local.service}/sentry-dsn"
     hub_environments                   = var.hub_environments
     hub_environments_legacy            = var.hub_environments_legacy
-    hub_config_host                    = "https://config-fargate.${local.hub_deployment}${var.hub_host}:443"
+    hub_config_host                    = "https://config-v2-fargate.${local.hub_deployment}${var.hub_host}:443"
```

Comment:
We will need to update hub_config_host in self-service's site.tf in verify-infrastructure-config to use config-v2-fargate.

Reply:
Have I taken care of that in https://github.com/alphagov/verify-infrastructure-config/pull/336 ?

Reply:
Yes.

Reply:
:+1: Thank you.